src/Actions/Actions/SaveAction.cs

```diff
@@ -24,15 +24,15 @@ public static class SaveAction
         /// <summary>
         /// Names for files inside of zipped snapshot
         /// </summary>
-        public const string elementFileName = "el.snapshot";
-        public const string screenshotFileName = "scshot.png";
-        public const string metadataFileName = "metadata.json";
-        public const string customPropsFileName = "CustomProperties.json";
+        public const string ElementFileName = "el.snapshot";
```

Comment:
This is a breaking change, no? Perhaps it would be better to accept a few `[SuppressMessage]` attributes here for the sake of backcompat

Reply:
(I didn't see any other instances of name changes for public things in the PR)

Reply:
These are only intended for `internal` use. Let me update the code accordingly...

Reply:
@dbjorge, I've pulled these `const` strings into a separate `internal` class that then gets used by both `LoadAction` and `SaveAction`